src/cluster_linearize.h

```diff
@@ -323,9 +331,21 @@ class LinearizationChunking
         Assume(subset.Any());
         Assume(subset.IsSubsetOf(m_todo));
         m_todo -= subset;
-        // Rechunk what remains of m_linearization.
-        m_chunks.clear();
-        BuildChunks();
+        if (GetChunk(0).transactions == subset) {
```

Comment:
re: `clusterlin: optimize rechunking in LinearizationChunking`

benchmark doesn't seem to show much improvement for this change. I understood this as purely an efficiency improvement which should be reflected in bench results?